assets/js/base/hooks/index.js

```diff
@@ -1,3 +1,5 @@
 export * from './use-query-state';
 export * from './use-shallow-equal';
 export * from './use-store-products';
+export * from './use-collection';
```

Comment:
These changes are already in master so I think the branch needs a sync.

Reply:
Yup, I split out the changes from here and was planning on syncing after the branch got merged.